src/main/kotlin/io/supabase/postgrest/builder/PostgrestBuilder.kt

```diff
@@ -31,7 +31,11 @@ open class PostgrestBuilder<T : Any> {
         this.jsonConverter = builder.jsonConverter
         this.schema = schema
 
-        builder.getSearchParams().forEach {
+        var params = builder.searchParams

```

Comment:
`searchParams` is not nullable, so the whole block can be simplified to

```
 builder.searchParams.forEach { (name, value) -> setSearchParam(name, value) }
```

The reason the tests are failing is the usage of `mockk` in the test which does not properly handle the builder.

This can be fixed by adjusting the PostgrestFilterBuilderTest and i.e. replacing the mockk call with

```
 private val postgrestBuilderMock = PostgrestBuilder<Any>(URI(""), mockk(), mockk(), emptyMap(), null)
```

Reply:
Cool, thanks.